tests/bluechi_machine_lib/bluechi_is_online.py

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright Contributors to the Eclipse BlueChi project
+#
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+import unittest
+import subprocess
+from typing import Tuple
+
+
+class BluechiIsOnline:
+    """
+    A Python module for the 'bluechi-is-online' command to check the status of agents, nodes, and the system.
+    """
+
+    def __init__(self, executable_path: str = "/usr/bin/bluechi-is-online"):
+        self.executable_path = executable_path
+
+    def _run_command(self, args: list) -> Tuple[int, str, str]:
```

Comment:
I would align this class with `BluechiCtl` class in `bluechi_machine_lib`, maybe even share the `run()` code with inheritance or something.